Gemfile.lock

```diff
@@ -172,6 +172,8 @@ GEM
     multipart-post (2.1.1)
     netrc (0.11.0)
     nio4r (2.5.8)
+    nokogiri (1.13.6-arm64-darwin)
+      racc (~> 1.4)
```

Comment:
Took a sec to spot the difference :-D  👍

Reply:
Yeah, I was confused at first.